ecms-social-integration/src/main/java/org/exoplatform/wcm/notification/plugin/FileActivityChildPlugin.java

```diff
@@ -264,5 +264,50 @@ private int getVersion(Node node) {
     }
     return values;
   }
+  private String[] getTitlesFromPath(Map<String, String> activityParams, String paramName) {
+    String[] values = null;
+    String value = activityParams.get(paramName);
+    if(value != null) {
+      values = value.split(FileUIActivity.SEPARATOR_REGEX);
+
+      for(int i =0 ; i<values.length; i++){
+        String str = values[i];
+        values[i] = str.substring(str.lastIndexOf('/')+1);
+      }
+
+    }   if (LOG.isDebugEnabled()) {
+      if(this.filesCount != 0 && (values == null || values.length != this.filesCount)) {
+        LOG.debug("Parameter '{}' hasn't same length as other activity parmameters", paramName);
+      }
+    }
+    return values;
+  }
+  private String[] getDocLinkbyNodeId(Map<String, String> activityParams, String paramName) {
+    String[] values = null;
+    String value = activityParams.get(paramName);
+    String workspace = activityParams.get(WORKSPACE);
+    if(value != null) {
+      values = value.split(FileUIActivity.SEPARATOR_REGEX);
 
-}
+      for(int i =0 ; i<values.length; i++){
+        String str = values[i];
+        values[i] = this.getBasePrivateRestUrl(workspace,str);
+      }
+    }
+
+    if (LOG.isDebugEnabled()) {
+      if(this.filesCount != 0 && (values == null || values.length != this.filesCount)) {
+        LOG.debug("Parameter '{}' hasn't same length as other activity parmameters", paramName);
+      }
+    }
+    return values;
+  }
+  public static String getBasePrivateRestUrl(String workspace,String id) {
+    return new StringBuffer(CommonsUtils.getCurrentDomain())
+            .append(PORTAL).append(PRIVATE_FOLDER_PATH)
```

Comment:
We need to use URL using document application with no Rest endpoint like in org.exoplatform.services.cms.documents.impl.DocumentServiceImpl#getDocumentUrlInSpaceDocuments